ValveResourceFormat/Resource/ResourceTypes/ModelAnimation/Animation.cs

```diff
@@ -52,7 +163,7 @@ public static IEnumerable<Animation> FromData(IKeyValueCollection animationData,
                 // Obviously we want to properly support animation decoding, but we are not there yet.
                 try
```

Comment:
This try/catch and the comment can be removed too now.

Reply:
Done, replaced with LINQ `.Select().ToArray()`